aiotg/bot.py

```diff
@@ -518,7 +519,13 @@ def __del__(self):
         await response.release()
 
     def _process_message(self, message):
-        chat = Chat.from_message(self, message)
+        chat_id = message["chat"]["id"]
+        chat = self._conversations.get(chat_id)
+
+        if chat:
+            chat.message = message # already exists is then updated
+        else:
+            chat = Chat.from_message(self, message) # otherwise, create new
```

Comment:
exactly the same as the original only if it is added to conversations the one that already exists is used